Cargo.toml

```diff
@@ -75,7 +75,7 @@ windows-sys = { version = "0.48", features = ["Win32_Security_Cryptography", "Wi
 [target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies]
 async-dnssd = "0.5"
 futures = "0.3"
-tokio = { version = "1.32", features = ["time", "rt"] }
+tokio = { version = "1.32", features = ["time", "rt", "rt-multi-thread"] }
```

Comment:
Since the feature is implied by `dns_resolver`, you can probably remove it from here too?

Reply:
`#[cfg(any(feature = "dns_resolver", target_os = "macos", target_os = "ios"))]`<-  you have to enable it for either `dns_resolver` or `macos`/`ios`, so it's required in both places

Reply:
Oh, okay! :+1: